doc/widgets/imageembedding.md

```diff
@@ -40,7 +40,17 @@ Image Embedding offers several embedders, each trained for a specific task. Imag
 Embedders
 ---------
 
-The **Inception v3** we are using is available [here](http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz). The only difference is that we are not using the final predictions, but the penultimate layer of the model, which represents images with vectors. A simple example of how to use the model can be found [here](https://github.com/tensorflow/models/blob/1af55e018eebce03fb61bba9959a04672536107d/tutorials/image/imagenet/classify_image.py). An [article](https://www.nature.com/articles/s41467-019-12397-x) by Godec et al. (2019) explains how the embeddings work and how to use it in Orange.
+**InceptionV3** is Google’s deep neural network for image recognition. It is trained on the ImageNet data set and performs with a 5.6 % top-5 error at the ImageNet dataset. The model we are using is available [here](http://download.tensorflow.org/models/image/imagenet/inception-2015-12-05.tgz). For the embedding, we use the activations of the penultimate layer of the model, which represents images with vectors. 
```

Comment:
> It is trained on the ImageNet data set and performs with a 5.6 % top-5 error at the ImageNet dataset.

--> It is trained on the ImageNet data set and performs with a 5.6 % top-5 error.

The second part is redundant in my opinion.

Reply:
Ok I can remove it.